validation/l2oo_test.go

```diff
@@ -19,33 +19,23 @@ import (
 	"github.com/ethereum/go-ethereum/ethclient"
 )
 
+type L2OOParams struct {
+	SubmissionInterval        *big.Int
+	L2BlockTime               *big.Int
+	FinalizationPeriodSeconds *big.Int
+}
+
 func TestL2OOParams(t *testing.T) {
 	isExcluded := map[uint64]bool{
-		999999999: true, // sepolia/zora    Incorrect submissionInterval, wanted 120 got 180
-		1740:      true, // sepolia/metal Incorrect submissionInterval
-		1750:      true, // mainnet/metal Incorrect submissionInterval
-		919:       true, // sepolia/mode Incorrect submissionInterval
-		8866:      true, // mainnet/superlumio Incorrect submissionInterval
-	}
-
-	checkEquality := func(a, b *big.Int) func() bool {
-		return (func() bool { return (a.Cmp(b) == 0) })
-	}
-
-	incorrectMsg := func(name string, want, got *big.Int) string {
-		return fmt.Sprintf("Incorrect %s, wanted %d got %d", name, want, got)
+		999999999: true, // sepolia/zora  Incorrect finalizationPeriodSeconds, 604800 is not within bounds [12 12]
```

Comment:
`finalizationPeriodSeconds` on Sepolia is an obvious place to look next. I suspect being strict about `12` is not what we want to do.

Reply:
Where is the `604800` value coming from?

Reply:
604800 is being used on sepolia/zora. It's one week's worth of seconds.